tests/regression/run_regression.py

```diff
@@ -0,0 +1,65 @@
+"""Checks export against known dataset.
+
+This is intended to be primarily run as part of CI only, and does not leverage
+pytest due to the requirements for AWS secrets/federated tokens/network 
+connectivity."""
+import os
+import sys
+
+from pathlib import Path
+
+from pandas import read_parquet
+
+references = os.listdir(f"{Path(__file__).resolve().parent}/reference")
+
+exports = os.listdir(f"{Path(__file__).resolve().parent}/data_export/core")
+
+if not references == exports:
+    ref_missing = []
+    for ref in references:
+        if ref not in exports:
+            ref_missing.append(ref)
+    export_missing = []
+    for export in exports:
+        if export not in references:
+            export_missing.append(export)
+    sys.exit(
+        "Found differences in files present:\n"
+        f"Files present in reference not in export: {str(ref_missing)}\n"
+        f"Files present in export not in reference: {str(export_missing)}"
+    )
+diffs = []
+for filename in references:
+    if filename.endswith(".parquet"):
+        ref_df = read_parquet(f"{Path(__file__).resolve().parent}/reference/{filename}")
```

Comment:
nit: maybe save these awkward paths in a variable at the top? You use them in two places, so just on safe side.

Reply:
yeah i buy it